### PR TITLE
genesis: Makes account balances format consistent across networks

### PR DESCRIPTION
**Pull request by evertonfraga**
This PR converts all balances from decimal to hex. Ordering shouldn't matter and the diff got rather impossible to review manually — but the tests cover that.

I put [this gist](https://gist.github.com/evertonfraga/62d8292e7f45bf1c9a2dcab4d28785d3) up for whoever wants to audit the process of data gathering and transformation. It was fun learning to use [jq](https://stedolan.github.io/jq/) to manipulate json content 😄 

To test:

1. run `npm run dist`
1. link/import this branch to `ethereumjs-vm/node_modules/ethereumjs-common`
1. from ethereumjs-vm dir, run `npm run build:dist && tape './tests/api/state/stateManager.js'`

[This test](https://github.com/ethereumjs/ethereumjs-vm/blob/f21d223a1f9293835ff9f23a08317f1d7324c1d0/tests/api/state/stateManager.js#L141-L158) ensures genesis block hash matches state root for each chain ([example](https://github.com/ethereumjs/ethereumjs-common/blob/master/src/chains/mainnet.json#L14)).

Related: https://github.com/ethereumjs/ethereumjs-vm/pull/589

Closes https://github.com/ethereumjs/ethereumjs-common/issues/35.

### CODE REVIEW

**Comment by evertonfraga** [comment]
mind note: I should add conformance tests to ensure new files adhere to hex.

File: src/genesisStates/kovan.json [hunk 1]
```diff
@@ -3,5 +3,5 @@
   "0x0000000000000000000000000000000000000002": "0x1",
   "0x0000000000000000000000000000000000000003": "0x1",
   "0x0000000000000000000000000000000000000004": "0x1",
-  "0x00521965e7bd230323c423d96c657db5b79d099f": "1606938044258990275541962092341162602522202993782792835301376"
+  "0x00521965e7bd230323c423d96c657db5b79d099f": "0x100000000000000000000000000000000000000000000000000"
 }
```
Comment (holgerd77) [inline src/genesisStates/kovan.json]: Ok.

File: src/genesisStates/ropsten.json [hunk 2]
```diff
@@ -1,259 +1,259 @@
...
+  "0x00000000000000000000000000000000000000f8": "0x0",
+  "0x00000000000000000000000000000000000000f9": "0x0",
+  "0x00000000000000000000000000000000000000fa": "0x0",
+  "0x00000000000000000000000000000000000000fb": "0x0",
+  "0x00000000000000000000000000000000000000fc": "0x0",
+  "0x00000000000000000000000000000000000000fd": "0x0",
+  "0x00000000000000000000000000000000000000fe": "0x0",
+  "0x00000000000000000000000000000000000000ff": "0x0",
+  "0x874b54a8bd152966d63f706bae1ffeb0411921e5": "0xc9f2c9cd04674edea40000000"
 }
```
Comment (holgerd77) [inline src/genesisStates/ropsten.json]: Ok, number of "1" -> "0x1" balances stayed the same, single more funded account moved to bottom.

File: src/genesisStates/kovan.json (hunk 1 above)
Comment (holgerd77) [reply src/genesisStates/kovan.json]: mainnet file comment (no way to attach found due to file size):

Consistency checks, addition/deletion number stays the same, random value tests (full text searches), genesis block hash test execution from Everton.

Looks good.

**Pull request merged**